src/button.cpp

```diff
@@ -51,15 +51,15 @@ bool Button::read( QTextStream &stream ) {
             else return false;
         }
         else if (*it == "layout") {
```

Comment:
Why is this not case insensitive now?

Reply:
Probably an oversight on my part, I'll fix it together with your other point.

Reply:
Done.